signer/src/message.rs

```diff
@@ -82,6 +82,27 @@ fn hash_message(msg: &wsts::net::Message, hasher: &mut sha2::Sha256) {
     }
 }
 
+impl SignerMessage {
+    pub fn random<R: rand::CryptoRng + rand::Rng>(rng: &mut R) -> Self {
```

Comment:
Ohh I think I see. This function and the other random functions are for the relay-client example __and__ for unit-tests. In which case, we can keep it here. If there are more functions that fall into this category, we can move all of them to their own feature gated module (like the `testing` module but with a `test-utils` feature or something).

Reply:
Yeah I originally had this under a #[cfg(test)] feature, but I found the functionality useful within the example code so I figured let's just expose it for now. I like the idea of feature gating it so that this doesn't bloat the delivered signer binary.

I think it's simpler to define this directly on the struct if you don't mind, so I'll add the feature attribute but I won't move it out for now.

Reply:
Done ✅